torch/_inductor/config.py

```diff
@@ -215,6 +215,10 @@ class cpp:
     # Valid values: "compile_error", "runtime_error", "accuracy"
     inject_relu_bug_TESTING_ONLY = None
 
+    # If None, autodetect whether or not AVX512/AVX2 can be used.  Otherwise,
+    # force usage as specified, without testing.
+    vec_isa_ok = None
```

Comment:
maybe worth a quick comment that the only reason for this config is that autodetect is slow? https://github.com/pytorch/pytorch/issues/100378

Reply:
Well I mean, you might also want to use this to just disable vectorization straight up, IDK.